EventFilter/RPCRawToDigi/interface/RPCLBPacker.h

```diff
@@ -0,0 +1,24 @@
+#ifndef EventFilter_RPCRawToDigi_RPCLBPacker_h
+#define EventFilter_RPCRawToDigi_RPCLBPacker_h
+
+#include <map>
+#include <vector>
+
+#include "CondFormats/RPCObjects/interface/RPCLBLink.h"
+#include "DataFormats/RPCDigi/interface/RPCDigiCollection.h"
+#include "EventFilter/RPCRawToDigi/interface/RPCLBRecord.h"
+
+class RPCInverseLBLinkMap;
+
+class RPCLBPacker
+{
+public:
+    /* https://twiki.cern.ch/twiki/bin/viewauth/CMS/DtUpgradeTwinMux#RPC_payload RPC_optical_links_data_format.pdf */
+    static void getRPCLBRecords(RPCInverseLBLinkMap const & _lb_map
```

Comment:
This will affect a major portion of the code.  I have seen the rule isn't enforced elsewhere, and the use here is obviously limited to local variables.  Do you really want this changed here, or is it a for-the-future comment?